tests/failpoints/cases/test_cmd_epoch_checker.rs

```diff
@@ -0,0 +1,364 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use engine_rocks::RocksSnapshot;
+use kvproto::raft_cmdpb::{RaftCmdRequest, RaftCmdResponse};
+use raft::eraftpb::MessageType;
+use raftstore::store::msg::*;
+use test_raftstore::*;
+use tikv_util::HandyRwLock;
+
+use std::sync::mpsc::{self, TryRecvError};
+use std::time::Duration;
+
+struct CbReceivers {
+    proposed: mpsc::Receiver<()>,
+    committed: mpsc::Receiver<()>,
+    applied: mpsc::Receiver<RaftCmdResponse>,
+}
+
+impl CbReceivers {
+    fn assert_not_ready(&self) {
+        self.proposed
+            .recv_timeout(Duration::from_millis(100))
+            .unwrap_err();
+        self.committed
+            .recv_timeout(Duration::from_millis(100))
+            .unwrap_err();
+        self.applied
+            .recv_timeout(Duration::from_millis(100))
+            .unwrap_err();
```

Comment:
Since we expect that these three channels can not receive anything 100ms latter, I think below code is better because original code needs 300ms.
```suggestion
        sleep_ms(100);
        assert_eq!(self.proposed.try_recv(), TryRecvError::Empty);
        assert_eq!(self. committed.try_recv(), TryRecvError::Empty);
        assert_eq!(self. applied.try_recv(), TryRecvError::Empty);
```